### PR TITLE
Feature/removeteam

### PR DESCRIPTION
**Pull request by panyachen**

### CODE REVIEW

**Comment by panyachen** [comment]
1. removed team member from visitor home page.
2. removed trial lessons from student 学员指南。
3. adding content in teacher dashboard.

**Comment by panyachen** [comment]
Adding shortform teacher manual to mspan platform.

File: app/views/teacher/home/pay.html.erb [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<h4>Payment</h4>
+
+<div class="spacer" style="height:20px;"></div>
```
Comment (rbaggio018) [inline app/views/teacher/home/pay.html.erb:3]: 这种inline css尽量避免。

**Review by rbaggio018 — APPROVED** [review]

**Comment by panyachen** [comment]
这么晚没睡觉？

**Comment by rbaggio018** [comment]
半夜起来送了下朋友

**Pull request merged**